content/applications/productivity/calendar/outlook.rst

```diff
@@ -136,3 +136,47 @@ The synchronization is a two-way process, meaning that events are reconciled in
 .. seealso::
    - :doc:`/applications/general/integrations/mail_plugins/outlook`
    - :doc:`google`
+
+Troubleshoot sync
+=================
+
+There may be times when the Microsoft Outlook calendar account does not sync correctly with Odoo.
+Sync issues can be seen in the database logs. In these cases, the account needs troubleshooting. A
+reset can be performed using the :guilabel:`Reset Account` button, which can be accessed by
+navigating to :menuselection:`Settings app --> Manage Users`. Then select the user to modify the
+calendar on, and click on the :guilabel:`Calendar` tab.
+
+.. image:: outlook/outlook-reset.png
+   :align: center
+   :alt: Reset buttons highlighted on the calendar tab of the user.
+
+Next, click :guilabel:`Reset Account` under the correct calendar.
+
+Reset options
+-------------
+
+The following reset options are available for troubleshooting Microsoft Outlook calendar sync with
```

Comment:
```suggestion
The following reset options are available for troubleshooting *Microsoft Outlook Calendar* sync with
```

Reply:
suggesting to capitalize *through* calendar because that's how Microsoft refers to it on their official website: https://support.microsoft.com/en-us/office/introduction-to-the-outlook-calendar-d94c5203-77c7-48ec-90a5-2e2bc10bd6f8